src/index.js

```diff
@@ -170,11 +170,14 @@ async function bundleMDX(
   }
 
   if (buildOptions.outdir && buildOptions.write) {
-    const code = await readFile(
-      path.join(buildOptions.outdir, '_mdx_bundler_entry_point.js'),
-    )
+    // @ts-ignore
```

Comment:
I don't like this but I couldn't see any other way. `buildOptions.entryPoints` can be `undfined` according to the type whilst we know that it isnt, it can also be a `Record` or `string[]`. All this added up to something that was hard to solve with JSDoc so I've set that line to ignore.

Reply:
Could we not add that to the if statement check?